homeassistant/components/duckdns/translations/en.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
```

Comment:
This isn't used (but should be used... 😄 )

Reply:
It is being used when a double entry is configured with the same domain.